HW2.ipynb

```diff
@@ -0,0 +1,94 @@
+# **Reference (source)**: 
```

Comment:
You cannot just add text to a Jupyter notebook. You have to add it from inside the Jupyter notebook server.